app/src/main/java/com/cortado/minionify/accumulators/Accumulator.java

```diff
@@ -0,0 +1,9 @@
+package com.cortado.minionify.accumulators;
+
+public interface Accumulator {
+    void set(String key, int value);
+    int get(String key);
+    boolean exists(String key);
+}
+
+class KeyDoesNotExists extends RuntimeException {}
```

Comment:
its unconventional declaring 2 classes in one file, and exceptions should always be final. what i meant when saying "declare it in the interface" is that you'd declare it in this way:

`int get(String key) throws KeyDoesNotExistException;`

so users would know that they can get this exception. move to another file, sorry for being unclear. btw rename to `KeyDoesNotExistException`

Reply:
Done. Thanks for the explanation.